sections/05_treat.md

```diff
@@ -224,12 +224,12 @@ how drugs are metabolized [@doi:10.1021/ci400518g].
 Here we primarily focus on ligand-based approaches that train on chemicals'
 features without requiring prior knowledge of the target. Chemical features may
 be represented as a list of molecular descriptors such as molecular weight, atom
-counts, charge representations, summaries of atom-atom relationships in the
+counts, functional groups, charge representations, summaries of atom-atom relationships in the
 molecular graph, and more sophisticated derived properties
 [@doi:10.1002/9783527628766].   Alternatively, chemicals can be characterized
 with the fingerprint bit vectors, textual strings, or novel learned
 representations described below. Neural networks have a long history in this
-domain [@tag:Baskin2015_drug_disc], and the 2012 Merck Molecular Activity
+domain [@tag:Baskin2015_drug_disc][@doi:10.1002/minf.201501008], and the 2012 Merck Molecular Activity
```

Comment:
Agreed, this is good to add.  We have these two and another two drug discovery-related reviews in the intro.

We can merge the refs: `[@tag:Baskin2015_drug_disc @doi:10.1002/minf.201501008]`